IBPSA/Fluid/HeatPumps/ModularReversible/Data/TableData2D/Generic.mo

```diff
@@ -0,0 +1,93 @@
+within IBPSA.Fluid.HeatPumps.ModularReversible.Data.TableData2D;
+record Generic "Partial two-dimensional data of refrigerant machines"
+    extends Modelica.Icons.Record;
+  parameter Real tabPEle[:,:]
+    "Electrical power consumption table, T in degC, Q_flow in W";
+  parameter Modelica.Units.SI.MassFlowRate mCon_flow_nominal
```

Comment:
Let's not use degC. Use K, otherwise it will be a disaster as users for sure mix up units.